engine/src/main/java/org/terasology/engine/core/modes/StateIngame.java

```diff
@@ -101,8 +109,20 @@ public Boolean get() {
     @Override
     public void dispose(boolean shuttingDown) {
         ChunkProvider chunkProvider = context.get(ChunkProvider.class);
+        AssetTypeManager assetTypeManager = context.get(ModuleAwareAssetTypeManager.class);
         chunkProvider.dispose();
 
```

Comment:
For readability, let's do this in blocks, i.e., first we dispose `chunkProvider`, then `assetTypeManager`, ...

```suggestion
        chunkProvider.dispose();

        AssetTypeManager assetTypeManager = context.get(ModuleAwareAssetTypeManager.class);
```

Alternatively, separate the context retrieval from the dispose actions with a new line.